src/Components/Receptions/Receptions.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import { DoctorContext } from '../../contexts/DoctorContext';
+import { getAll, getDoctors } from '../../services/api';
+import Error from '../AlertError/Error';
+import HeaderComponent from '../Header/Header';
+import List from '../List';
+import './index.css';
+
+const Receptions = () => {
+  const [deleteItem, setDeleteItem] = useState(null);
+  const [idToDelete, setIdToDelete] = useState(null);
+  const [editItem, setEditItem] = useState(null);
+  const [idToEdit, setIdToEdit] = useState(null);
+  const [receptions, setReceptions] = useState([]);
+  const [doctors, setDoctors] = useState([]);
+  const [filtered, setFiltered] = useState([]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const allReceptions = getAll();
+    allReceptions
+      .then((data) => {
+        setReceptions(data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+
+    const doctors = getDoctors();
+    doctors
+      .then((data) => {
+        setDoctors(data);
+      });
+  }, []);
+
+  const setAfterDelete = (id) => {
+    setIdToDelete(id);
+    setDeleteItem(true);
+  };
+  const setAfterEdit = (id) => {
+    setIdToEdit(id);
+    setEditItem(true);
```

Comment:
why do you need second state if you can identify if there's something to edit by idToEdit state